support-frontend/typescript-errors.json

```diff
@@ -4085,6 +4103,14 @@
         {
             "path": "assets/helpers/tracking/googleTagManager.ts",
             "message": "Could not find a declaration file for module 'uuid'. 'support-frontend/node_modules/uuid/index.js' implicitly has an 'any' type."
+        },
+        {
+            "path": "assets/pages/contributions-landing/components/ContributionFormContainer.tsx",
+            "message": "Could not find a declaration file for module 'react-router-dom'. 'support-frontend/node_modules/react-router-dom/index.js' implicitly has an 'any' type."
+        },
+        {
+            "path": "assets/pages/contributions-landing/contributionsLanding.tsx",
+            "message": "Could not find a declaration file for module 'react-router-dom'. 'support-frontend/node_modules/react-router-dom/index.js' implicitly has an 'any' type."
```

Comment:
I think running `yarn` will probably fix up these ones. I think you'll then need to manually run `yarn ts-errors` to regenerate this file

Reply:
(might also need to rebase/merge main first too)